spec.html

```diff
@@ -39056,10 +38793,10 @@ <h1>GeneratorResume ( _generator_, _value_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-generatorresumeabrupt" aoid="GeneratorResumeAbrupt">
-        <h1>GeneratorResumeAbrupt ( _generator_, _abruptCompletion_ )</h1>
-        <p>The abstract operation GeneratorResumeAbrupt takes arguments _generator_ and _abruptCompletion_. It performs the following steps when called:</p>
+        <h1>GeneratorResumeAbrupt ( _generator_, _abruptCompletion_, _generatorBrand_ )</h1>
+        <p>The abstract operation GeneratorResumeAbrupt takes arguments _generator_, _abruptCompletion_ (a completion record whose [[Type]] is ~return~ or ~throw~), and _generatorBrand_. It performs the following steps when called:</p>
```

Comment:
```suggestion
        <p>The abstract operation GeneratorResumeAbrupt takes arguments _generator_, _abruptCompletion_ (a Completion Record whose [[Type]] is ~return~ or ~throw~), and _generatorBrand_. It performs the following steps when called:</p>
```

(We're somewhat inconsistent about capitalization, but "Completion Record" is the more common form.)